nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java

```diff
@@ -787,46 +763,46 @@ protected boolean scheduleAfterCommit(Work work, Scheduling scheduling) {
             transactionManager = null;
         }
         if (transactionManager == null) {
-            if (log.isDebugEnabled()) {
-                log.debug("Not scheduling work after commit because of missing transaction manager: " + work);
+            if (getLog().isDebugEnabled()) {
```

Comment:
I am fine with this getLog() accessor for error or log during init shutdown, 
but here it is used many times per work log, I prefer a private static final